game_api.txt

```diff
@@ -1115,3 +1115,26 @@ This function registers a shapeless recipe that takes `ingredient`
 and `result` as input and outputs `result`.
 
 The metadata of the input `result` is copied to the output `result`.
+
+
+Log API
+-------
+
+Logs action of the player with a node at a certain position.
+By default only actions of real players are logged.
```

Comment:
Please document that players are considered "fake" only if `player.is_fake_player` is truthy.

Reply:
done